docs/sql-migration-guide.md

```diff
@@ -54,6 +54,8 @@ license: |
   
   - In Spark 3.1, creating or altering a view will capture runtime SQL configs and store them as view properties. These configs will be applied during the parsing and analysis phases of the view resolution. To restore the behavior before Spark 3.1, you can set `spark.sql.legacy.useCurrentConfigsForView` to `true`.
 
+  - In Spark 3.1, CHAR/CHARACTER and VARCHAR types become individual types from string. By default, they can only be used in table schema, not functions/operators. To restore the behavior before Spark 3.1, where treats them as string with length parameter simply ignored, you can set `spark.sql.legacy.charVarcharAsString` to `true`.
```

Comment:
how about `CHAR/CHARACTER and VARCHAR types become individual types from string` -> `we support CHAR/CHARACTER and VARCHAR types in our type system framework instead of replacing them with STRING types`?

Reply:
nit: `table schema` -> `a table schema`

Reply:
`, where treats them as string with length parameter simply ignored,` -> `, which treats them as STRING types and ignores a length parameter (e.g., CHAR(4)), `?

Reply:
thanks, updated